src/utils/settingsValidation/__tests__/splitFilters.spec.ts

```diff
@@ -91,3 +92,45 @@ describe('validateSplitFilters', () => {
   });
 
 });
+
+
+describe('validateSetFilter', () => {
```

Comment:
Since we are validating the function called `validateSplitFilters` (`validateSetFilter` doesn't exist), this test should be inside `describe('validateSplitFilters', () => {` like the other ones.

Something like:

```
describe('validateSplitFilters', () => {
  ...

  test('validates flag set filters', () => {
    const getOutput = (testIndex: number) => {
    // THE REST OF THE TEST CODE
  });

});
```